### ISSUE TITLE
Apicurio registry SQL version ENTRYPOINT ["<path/Script.sh>"]

### ISSUE THREAD

**Issue by ozarkblue** [Apicurio/apicurio-registry#2658]
I'm from a very large org and we're trying to perform a POC to see if we can use Apicurio Registry SQL version for our event schema repo.
As part of this POC I'm trying to install latest registry image in our org kubernetes with a Dockerfile

I'm seeing below error in deployment pod container log  and its failing to install:
 "standard_init_linux.go:228: exec user process caused: permission denied "

I've downloaded the Apicurio registry SQL image latest version from dockerhub (path given in Apicurio image download link) and using this with below Dockerfile

FROM <Our Org internal network image repo path where I downloaded the image first from dockerhub>:latest
WORKDIR /app
COPY . .
USER root
RUN adduser --system --uid 10000 --shell /sbin/nologin --home /opt/app/ app
RUN chmod 755 *.*
USER 10000

To avoid above exec permission error I've tried adding  entrypoint below at end of dockerfile (eeing this in github test dockerfile of apicurio registry) but that ended with error that this file is not found in image.

ENTRYPOINT ["pr_check_docker.sh"]

Please suggest Entrypoint file and path for Dockerfile or suggest a solution for above exec permission error.
 

Note - The same image works when I install in my local desktop docker with docker run command and environment file
docker run --env-file ./.env -p 0.0.0.0:8080:8080 --expose 8080 --network apicurio-network 0498ae15b5f9

**Comment by carlesarnal** [comment]
HI,

The error might be caused because you're using the wrong image. We no longer push `latest` as the tag. You must use either `latest-snapshot` (with the latest changes from master) or `latest-release`. Please, try one of those and let me know if you still have issues.

**Comment by ozarkblue** [comment]
This issue is resolved now.
It was due to the fact that non root user  that I was making (USER 10000 in above script) doesn't have permission to run the image.
When I directly used the image in my deplymement yml (skipping build step)it worked.
No entrypoint required to provide and I removed the Dockerfile that was used to build(download image and create use basically).


I had to provide below to download and deploy image.

Here is part of my YML.

build:
  skip: true
  artifact: apicurio_poc_sql
  buildType: docker
  docker:
    app:
      dockerFile: Dockerfile

deploy:
  helm:
    values:
      container:
        image: apicurio/apicurio-registry-sql
        tag: 2.2.5.Final
        repository: <our org proxy image path to docker hub>
